client/components/Widget.jsx

```diff
@@ -10,7 +11,7 @@ class Widget extends React.Component {
   }
 
   handleClick() {
```

Comment:
What does this function do? I don't see any code in its body and It doesn't like like its just contracted lol.

Reply:
you asked this last time haha.  it's just for testing purposes right now so I have coverage on my button element.  will probably just do a dummy log in the end or some other use when interacting on the site since we wont have checkout.